porcupine/plugins/directory_tree.py

```diff
@@ -410,7 +268,12 @@ def _open_and_refresh_directory(self, dir_path: Path, dir_id: str) -> None:
         for child_path, child_id in path2id.items():
             self._update_tags_and_content(project_root, child_id)
 
-        for index, child_id in enumerate(sorted(self.get_children(dir_id), key=self._sorting_key)):
+        for index, child_id in enumerate(
+            sorted(
+                self.get_children(dir_id),
+                key=(lambda item_id: [f(item_id) for f in self.sorting_keys]),
```

Comment:
Output of sorting keys may depend on git tags which are set afterwards. Can this cause wrong order bug?